### PR TITLE
Add LEX space

### PR DESCRIPTION
**Pull request by Ro5s**
**Add space for LexDAO Engineering - $LEX Token**  
  
Etherscan: https://etherscan.io/address/0xA5C5C8Af327248c4c2dce810a3d3Cffb8C4F66ab#code
Website: http://lexdao.org/
Twitter: https://twitter.com/lex_DAO

### CODE REVIEW

File: spaces/lex/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.lex {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (e-asia) [inline spaces/lex/skin.scss:10]: To use the default theme you will want to delete skin.scss from your space folder.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/lex/skin.scss (hunk 1 above)
Comment (Ro5s) [reply spaces/lex/skin.scss:10]: oh, thanks for clarification! Will delete file.

**Comment by Ro5s** [comment]
thanks for the review @e-asia. I have removed the extra skin.scss file in order to get the default space theme.

File: spaces/lex/index.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "key": "$LEX",
```
Comment (e-asia) [inline spaces/lex/index.json:2]: The key will be the url and we want it all in lowercase and without special characters. Please modify the value of the key by something like `lex`. Thank you.
```suggestion
  "key": "lex",
```

**Review by e-asia — CHANGES_REQUESTED** [review]

**Comment by Ro5s** [comment]
ah, thanks for that catch @e-asia . I have committed your suggested change for url.

**Review by e-asia — APPROVED** [review]

**Pull request merged**